src/magic.h

```diff
@@ -68,6 +69,7 @@ enum valid_target {
     target_ally,
     target_hostile,
     target_self,
+    target_monster_id,
```

Comment:
i'm not convinced it's necessary or the best thing to do to have to specify a valid_target if you have a monster_id already. if you _don't_ use this, it could potentially allow for slightly more flexible targets